.github/workflows/push.yml

```diff
@@ -199,3 +196,52 @@ jobs:
             all-build-
       - name: Roave BC Check
         uses: docker://nyholm/roave-bc-check-ga
+
+  profile:
+    runs-on: ubuntu-latest
+    name: profile
+    needs: [setup, phpunit]
+    services:
+      blackfire:
+        image: blackfire/blackfire
+        ports:
+          - 8707:8707
+        env:
+          BLACKFIRE_SERVER_ID: "517a7aa6-1ec8-4d29-bff8-dddecf4333de"
+          BLACKFIRE_SERVER_TOKEN: "21795bdce7c0b5d24f0ccbb42e2a7518feb5359840752b163652899f927cbf2b"
```

Comment:
You should **never** expose your blackfire credentials (either client or server). This should end up with other secrets.